pyproject.toml

```diff
@@ -2,4 +2,4 @@
 profile = "black"
 
 [tool.djlint]
-ignore="H021,H026"
+ignore="H021,H025,H026"
```

Comment:
What is H025 and why are we ignoring now?

Reply:
H025 is an orphaned tag one.  With one of the recent updates to djlint, it's marking a few as orphaned even though they're actually fine; Those ones are concatenating strings together to make a complete`<a>` tag but the linter isn't recognising it